CefSharp.Core.Runtime/CefSharp.Core.Runtime.vcxproj

```diff
@@ -267,6 +267,8 @@
     <ClInclude Include="Cef.h" />
     <ClInclude Include="DragData.h" />
     <ClInclude Include="Internals\CefDevToolsMessageObserverAdapter.h" />
+    <ClInclude Include="Internals\CefMediaAccessCallbackWrapper.h" />
```

Comment:
The new headers need to be added to the netcore `vcxproj` also. (use the VS to do this)